networking/metallb/metallb-operator-install.adoc

```diff
@@ -13,6 +13,17 @@ The Operator starts MetalLB in the same namespace that the Operator is installed
 
 MetalLB and IP failover are incompatible. If you configured IP failover for your cluster, perform the steps to xref:../../networking/configuring-ipfailover.adoc#nw-ipfailover-remove_configuring-ipfailover[remove IP failover] before you install the Operator.
 
+[NOTE]
+====
+To ensure BGP and BFD metrics appear in Prometheus, you can label the namespace as in the following example:
```

Comment:
Are these initialisms recognizable enough to not need to be expanded?

Is this really the best place for this note - it seems like it should go in the procedure where you're using this namespace. (And I'm not 100% sure which namespace you're talking about.)

Reply:
Yeah, putting this Note in the procedure step is probably better. I'm not sure about recognizability of the initialism...BGP and BFD are not mentioned otherwise in this assembly, but they are nearby in next steps. In the 'About MetalLB and the MetalLB Operator' section just above the install, the initials are not expanded either, only in the Configurations is it expanded.

Reply:
@kalexand-rh @opayne1 can you take another 👀 at this?